### PR TITLE
WinUI-2685 : Update RadialGague UG content for MarkerPointer breaking…

### PR DESCRIPTION
**Pull request by VishalOmprasad**
Updated the UG document content regarding Split MarkerPointer into ShapePointer and ContentPointer.

### CODE REVIEW

File: winui/Radial-Gauge/content-pointer.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+title: Marker Pointer in WinUI Radial Gauge control | Syncfusion
+description: Learn here all about Marker Pointer feature of Syncfusion WinUI Radial Gauge control with image, text pointer support and more.
+platform: WinUI
+control: SfRadialGauge
+documentation: ug
+---
+
+# Shape Pointer in WinUI Radial Gauge
+
+The `ContentPointer` in `SfRadialGauge` allows you to use any content using the `Content` property as a pointer to mark a specified value.
```
Comment (vigneshrameshsync) [inline winui/Radial-Gauge/content-pointer.md:12]: We have API link for RadialGauge. Please include it.

File: winui/Radial-Gauge/content-pointer.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title: Marker Pointer in WinUI Radial Gauge control | Syncfusion
+description: Learn here all about Marker Pointer feature of Syncfusion WinUI Radial Gauge control with image, text pointer support and more.
+platform: WinUI
+control: SfRadialGauge
+documentation: ug
+---
+
+# Shape Pointer in WinUI Radial Gauge
```
Comment (vigneshrameshsync) [inline winui/Radial-Gauge/content-pointer.md:10]: Check title

File: winui/Radial-Gauge/content-pointer.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title: Marker Pointer in WinUI Radial Gauge control | Syncfusion
+description: Learn here all about Marker Pointer feature of Syncfusion WinUI Radial Gauge control with image, text pointer support and more.
```
Comment (vigneshrameshsync) [inline winui/Radial-Gauge/content-pointer.md:4]: Check title and description.

File: winui/Radial-Gauge/content-pointer.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+
+this.Content = sfRadialGauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![WinUI Radial Gauge Content Pointer](images/content-pointer/default-content-pointer.png)
+
+## Add Image to Content Pointer
```
Comment (vigneshrameshsync) [inline winui/Radial-Gauge/content-pointer.md:73]: It is just the title. So avoid unwanted case changes.

File: winui/Radial-Gauge/content-pointer.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+
+{% endtabs %}
+
+![WinUI Radial Gauge Content Pointer with Image](images/content-pointer/winui-radial-gauge-image-pointer.png)
+
+## Content position customization
+
+The content pointer can be moved near or far from its actual position using the `MarkerOffset` and `OffsetUnit` properties. 
+
+When you set `OffsetUnit` to pixel, the content pointer will be moved based on the pixel value. If you set `OffsetUnit` to factor, then provided factor will be multiplied with the axis radius value, and then the pointer will be moved to corresponding value. The default value of `OffsetUnit` is `SizeUnit.Pixel`.
```
Comment (vigneshrameshsync) [inline winui/Radial-Gauge/content-pointer.md:140]: We have API links for OffsetUnit and SizeUnit.

File: winui/Radial-Gauge/content-pointer.md [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+            </gauge:RadialAxis.Pointers>
+        </gauge:RadialAxis>
+    </gauge:SfRadialGauge.Axes>
+</gauge:SfRadialGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfRadialGauge sfRadialGauge = new SfRadialGauge();
```
Comment (vigneshrameshsync) [inline winui/Radial-Gauge/content-pointer.md:173]: Align the code properly.

**Review by vigneshrameshsync — CHANGES_REQUESTED** [review]

File: winui/Radial-Gauge/content-pointer.md (hunk 1 above)
Comment (VishalOmprasad) [reply winui/Radial-Gauge/content-pointer.md]: API link for SrRadialGauge, added.

File: winui/Radial-Gauge/content-pointer.md (hunk 2 above)
Comment (VishalOmprasad) [reply winui/Radial-Gauge/content-pointer.md]: Changed the title.

File: winui/Radial-Gauge/content-pointer.md (hunk 3 above)
Comment (VishalOmprasad) [reply winui/Radial-Gauge/content-pointer.md]: Changes the title and description.

File: winui/Radial-Gauge/content-pointer.md (hunk 4 above)
Comment (VishalOmprasad) [reply winui/Radial-Gauge/content-pointer.md]: Removed unwanted case changes.

File: winui/Radial-Gauge/content-pointer.md (hunk 5 above)
Comment (VishalOmprasad) [reply winui/Radial-Gauge/content-pointer.md]: API link for OffsetUnit and SizeUnit was added.

File: winui/Radial-Gauge/content-pointer.md (hunk 6 above)
Comment (VishalOmprasad) [reply winui/Radial-Gauge/content-pointer.md:173]: Alignment was done proper.

**Review by vigneshrameshsync — APPROVED** [review]
The changes are fine.

**Pull request merged**